src/ui/public/doc_table/lib/get_sort.js

```diff
@@ -6,7 +6,7 @@ import _ from 'lodash';
  * @param {object} indexPattern used for determining default sort
  * @returns {object} a sort object suitable for returning to elasticsearch
  */
-export function getSort(sort, indexPattern) {
+export function getSort(sort, indexPattern, sortOptions = {}) {
```

Comment:
How about just passing `defaultOrder` here, since that is the only property in use right now?

Reply:
👍 https://github.com/elastic/kibana/pull/13708/commits/faba375bd9789e5e40c5be031d7491bee332d04a